solr/CHANGES.txt

```diff
@@ -133,7 +133,7 @@ Dependency Upgrades
 
 Other Changes
 ---------------------
-(No changes)
+* SOLR-17399: Replace the use of the deprecated java.util.Locale constructor with Locale Builder API
```

Comment:
I often just put the proposed changelog text in my PR, because I always end up with conflicts in this file ;-).  Then I make the change right at the very end!